src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -172,6 +145,7 @@ const Query: FC<PipeProp> = ({Context}) => {
       updateText,
       editorContext.editor,
       variables,
+      launcher,
```

Comment:
Was having a memoize issue for the launcher not grabbing the latest `showId`. Hence, these minor changes for the launcher.